### ISSUE TITLE
feature request; find holiday by name

### ISSUE THREAD

**Issue by cip123** [commenthol/date-holidays#482]
I apologize if that's already in the library and I couldn't find it, but I would like to be able to find, for example, the Thanksgiving date in different countries.

The problem is that currently, the holiday is called "Thanksgiving Day" in the US and "Thanksgiving" in Canada.  So in order to find the Thanksgiving date, I have to know in advance the name in different countries.

Is there a way to get those dates just by passing in the country and the name "Thanksgiving"?

**Comment by commenthol** [comment]
No. You'd need to search for the holiday using the data as there is no database to query from.
